server/memo.go

```diff
@@ -60,7 +60,26 @@ func (s *Server) registerMemoRoutes(g *echo.Group) {
 	})
 
 	g.GET("/memo", func(c echo.Context) error {
-		userID := c.Get(getUserIDContextKey()).(int)
+		userID, ok := c.Get(getUserIDContextKey()).(int)
+		if !ok {
+			if c.QueryParam("userID") != "" {
+				var err error
+				userID, err = strconv.Atoi(c.QueryParam("userID"))
```

Comment:
```suggestion
				userID, err := strconv.Atoi(c.QueryParam("userID"))
```

Reply:
I can't modify it like this, the userID will be overwritten in the if area

Reply:
Sorry, my bad